schemas/f142_logdata.fbs

```diff
@@ -62,6 +62,7 @@ table LogData {
 	source_name: string; // identify source on multiplexed topics, e.g. PV name if from EPICS
 	value: Value;        // may be scalar or array
 	timestamp: ulong;    // nanoseconds past epoch (1 Jan 1970), zero reserved for invalid timestamp
+	expires_after: ulong;  // optional, nanoseconds after the timestamp when this value is expired
```

Comment:
I think you need to add this as the last field for it to be a non-breaking change to the schema, otherwise consumers wanting to access `fwdinfo` would get unexpected results.